TAScheduler/views_methods.py

```diff
@@ -404,6 +404,23 @@ def courseTAAsmgt(self, active_ta, active_course):  # new remove sectionTAAsmgt
             raise RuntimeError("Instructor is already assigned to max number of course permitted")
         TAToCourse.objects.create(ta=active_ta.database, course=active_course.database)
 
+    def editMyContactInfo(self, **kwargs):
```

Comment:
Does this method work? If so great. Otherwise you might have to go through each input individually